Source/WebCore/html/HTMLCanvasElement.cpp

```diff
@@ -433,18 +433,20 @@ WebGLRenderingContextBase* HTMLCanvasElement::createContextWebGL(WebGLVersion ty
 
     // TODO(WEBXR): ensure the context is created in a compatible graphics
     // adapter when there is an active immersive device.
-    m_context = WebGLRenderingContextBase::create(*this, attrs, type);
-    if (m_context) {
+    auto context = WebGLRenderingContextBase::create(*this, attrs, type);
+    WeakPtr weakContext = context.get();
```

Comment:
Can this use `RefPtr` instead?

Reply:
WebGLRenderingContextBase is not refcounted. create() returns a unique_ptr.